Automattic/jetpack
# Instant Search: Simplify modal animation

Fixes https://github.com/Automattic/jetpack/issues/19836.

#### Changes proposed in this Pull Request:

* Drop animation from modal opening and use a simple opacity transition instead for a snappier feel.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
On a test site with a Jetpack Search plan, start a search to open the modal. Verify that there's a short transition but no slide animation. When closing the modal, verify that there is no transition or animation.

![2021-06-04 12-16-00 2021-06-04 12_16_48](https://user-images.githubusercontent.com/17325/120727552-ed940380-c52e-11eb-8a28-412c13f3df0a.gif)

cc @pablohoneyhoney @evilluendas
